app/models/application_form.rb

```diff
@@ -72,7 +72,25 @@ def equality_and_diversity_answers_provided?
   attribute :recruitment_cycle_year, :integer, default: -> { RecruitmentCycle.current_year }
 
   before_create -> { self.support_reference ||= GenerateSupportRef.call }
-  after_save -> { application_choices.update_all(updated_at: Time.zone.now) }
```

Comment:
If we want touches on application form to also touch the application choices as we wanted before (e.g. change WorkHistory > Touch ApplicationForm > Touch ApplicationChoice), we need to put in an `after_touch` since `after_save` is not triggered on touch